README.md

```diff
@@ -137,6 +137,20 @@ The options can be specified in the `package.json` file under the `react-native-
 
 The name of the folder with the source code which should be compiled. The folder should include an `index` file.
 
+#### `exclude`
```

Comment:
Move it to after `output`, since the `source` and `output` options are related to each other and it's better to have them closer.